apps/web/package.json

```diff
@@ -47,6 +47,7 @@
     "clsx": "^2.1.0",
     "dayjs": "^1.11.10",
     "drizzle-orm": "^0.29.4",
+    "epub-gen-memory": "^1.0.10",
```

Comment:
This library seems unmaintained with last version being from 2 years ago. I wouldn't want to add this as a dependency to be honest. Is there a better library to use for the epub generation?

It's also worth noting that this library "fetches" links (e.g the images in a URL) from the `web` container which is something we currently never do. Not sure how I feel about this. It might be better if we do this from the user's browser instead given that we have all the info there anyways.

Reply:
If we do it from the users browsers, we will have CORS issues. I created a chrome extension for games workshop before, where this was a huge issue and it was basically missing all the images all the time. If we don't include all the images, this will be the first bug report we have, that there are not all images included.

Regarding the library, there is https://github.com/cyrilis/epub-gen, which seems quite popular, but has not been updated for 6 years. There is a fork https://github.com/lesjoursfr/html-to-epub, which is still maintained. If you prefer this one, I can try using it instead.

Reply:
There is also a fork from the current repo: https://www.npmjs.com/package/@epubkit/epub-gen-memory

Reply:
I have switched to the epubkit generation, as this generates in memory instead of writing to the disk and it is maintained and used in the tool https://epubkit.app/